torch/onnx/_internal/exporter.py

```diff
@@ -772,6 +781,54 @@ def save(
             else:
                 serializer.serialize(self, destination)
 
+    @_beartype.beartype
+    def save_sarif_log(self, destination: str) -> None:
```

Comment:
```suggestion
    def save_sarif(self, destination: str) -> None:
```
Rename for discussion, only.
Is it implied that SARIF is always a log file? If so, we can save the `_log` suffix. Or is there `sarif_another_type_of_artifact` possible in the sarif world?

Reply:
I think your suggestion is more concise.

Reply:
i like `save_sarif` better because whoever doesnt know what SARIF is, has no reason to `save_sarif`
However, who does, may not know they can get the sarif from `save_diagnostics`.

Reply:
`save_diagnostics_to_sarif` then?